cmd/nerdctl/builder_build_test.go

```diff
@@ -253,15 +247,10 @@ COPY %s /`,
 	assert.Equal(t, string(data), testContent)
 }
 
-func createBuildContext(dockerfile string) (string, error) {
-	tmpDir, err := os.MkdirTemp("", "nerdctl-build-test")
-	if err != nil {
-		return "", err
-	}
-	if err = os.WriteFile(filepath.Join(tmpDir, "Dockerfile"), []byte(dockerfile), 0644); err != nil {
-		return "", err
-	}
-	return tmpDir, nil
+func createBuildContext(t *testing.T, dockerfile string) (string, error) {
+	tmpDir := t.TempDir()
+	err := os.WriteFile(filepath.Join(tmpDir, "Dockerfile"), []byte(dockerfile), 0644)
+	return tmpDir, err
```

Comment:
No need to return err

Reply:
I've moved the error assertion directly to the function to remove it from returned values